src/components/v2v3/V2V3Project/V2V3PayButton/V2V3ConfirmPayModal/V2V3ConfirmPayModal.tsx

```diff
@@ -135,10 +140,18 @@ export function V2V3ConfirmPayModal({
     } = form.getFieldsValue()
 
     const txBeneficiary = beneficiary ?? userAddress
-    const delegateMetadata = encodeJB721DelegatePayMetadata({
-      ...(payProjectForm?.payMetadata as JB721DelegatePayMetadata),
-      ...payMetadataOverrides(projectId),
-    })
+
+    const delegateMetadata =
+      nftContractVersion === '1' // old delegate v1
```

Comment:
**suggestion** pretty sure there is a constant for `'1'` right?

Reply:
Yep, there is. Thanks